diff --git a/src/process/index.js b/src/process/index.js
index f445024..2f4adcf 100644
--- a/src/process/index.js
+++ b/src/process/index.js
@@ -5,8 +5,22 @@ import fs from 'node:fs';
 import { dirname, join } from 'path';
 import { fileURLToPath } from 'url';
 
-const __dirname = dirname(fileURLToPath(import.meta.url));
-const DetectableDB = JSON.parse(fs.readFileSync(join(__dirname, 'detectable.json'), 'utf8'));
+//const __dirname = dirname(fileURLToPath(import.meta.url));
+//const DetectableDB = JSON.parse(fs.readFileSync(join(__dirname, 'detectable.json'), 'utf8'));
+
+const DetectableDB = require('./detectable.json');
+
+DetectableDB.push(
+  {
+    aliases: ["Deadlock"],
+    executables: [
+      { is_launcher: false, name: "win64/deadlock.exe", os: "win32" },
+    ],
+    hook: true,
+    id: "1276737795012165766",
+    name: "Deadlock"
+  }
+);
 
 import * as Natives from './native/index.js';
 const Native = Natives[process.platform];
